accord-core/src/main/java/accord/local/Node.java

```diff
@@ -241,7 +242,21 @@ public long epoch()
     private synchronized EpochReady onTopologyUpdateInternal(Topology topology, boolean startSync)
     {
         Supplier<EpochReady> bootstrap = commandStores.updateTopology(this, topology, startSync);
-        return this.topology.onTopologyUpdate(topology, bootstrap);
+        Supplier<EpochReady> ordering = () -> {
```

Comment:
this is one of the two core changes in this PR.  By ordering `EpochReady.coordination`, we have the property that epoch=5 being ready implies all epochs before are also, this property is required for C* to recover the known epochs learned *before* joining the ring.